### DISCUSSION TITLE
Adding bounding surface plasticity

### DISCUSSION THREAD

**Discussion by zzgator4real** [GEOS-DEV/GEOS#2244] [Q&A]
Dear developers,

Can I add a new material model, e.g., bounding_surface_cam_clay.cpp and .hpp into /src/coreComponents/constitutive/solid? Do I need extra linking or compiling procedures?

Regards,

Zhe

**Comment by joshua-white** [comment]
@zzgator4real, as long as your implementation conforms to our existing `solidModel` interface, there should be no problem bringing this model in.  I would mimic the existing `modifiedCamClay` implementation to the extent possible as a template for your implementation.  Let us know if you need assistance.
